charts/victoria-metrics-cluster/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,51 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "victoria-metrics.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+  {{- include "victoria-metrics.common.metaLabels" . | nindent 4 }}
+  {{- if .Values.rbac.extraLabels }}
+  {{ toYaml .Values.rbac.extraLabels | indent 4}}
+  {{- end }}
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default'

```

Comment:
does user need to add more annotation here which can be specified in values.yaml?

Reply:
Not sure. We do not but someone might. I have added it now.